HealthSDK/GiniHealthSDK/Documentation/source/Integration.md

```diff
@@ -147,29 +96,132 @@ dispatchGroup.notify(queue: .main) {
+public func paymentView(documentId: String) -> UIView
 ```
 
 > - We suggest placing this `UIView` within a vertical `UIStackView`. Additionally, in the `prepareForReuse()` function of each cell, remove the payment component view if it exists.
 > - Furthermore, employing automatic dimension height in the `UITableView` containing the cells is recommended.
 
-#### 5. `PaymentComponentsController` has 2 delegates that you can listen to: `PaymentComponentsControllerProtocol` and the `PaymentComponentViewProtocol`
-
-* `PaymentComponentsControllerProtocol` provides information when the `PaymentComponentsController` is loading. You can show/hide an `UIActivityIndicator` based on that.
-* `PaymentComponentsControllerProtocol` provides completion handlers when `PaymentComponentsController` fetched successfully payment providers or when it failed with an error.
-
 * `PaymentComponentViewProtocol` is the view protocol and provides events handlers when the user tapped on various areas on the payment component view (more information icon, bank/payment provider picker, the pay invoice button and etc.).
 
 > - Make sure you properly link these delegates to get notified.
 
-<br>
-<center><img src="img/Customization guide/PaymentComponentScreen.PNG" height="500"/></center>
-</br>
+## Show PaymentInfoViewController
+
+The `PaymentInfoViewController` displays information and an FAQ section about the payment feature.
+It requires a `PaymentComponentsController` instance (see `Integrate the Payment component` step 1).
+
+> **Note:** 
+> - The `PaymentInfoViewController` can be presented modally, used in a container view or pushed to a navigation view controller. Make sure to add your own navigational elements around the provided views.
+
+> **Important:** 
+> - The `PaymentInfoViewController` presentation should happen in `func didTapOnMoreInformation(documentId: String?)` inside
+`PaymentComponentViewProtocol` implementation.(`Integrate the Payment component` step 3).
+
+```swift
+func didTapOnMoreInformation(documentId: String?) {
+    let paymentInfoViewController = paymentComponentsController.paymentInfoViewController()
+    self.yourInvoicesListViewController.navigationController?.pushViewController(paymentInfoViewController,
+                                                                                 animated: true)
+}
+ ```
+
+## Show BankSelectionBottomSheet
+
+The `BankSelectionBottomSheet` displays a list of available banks for the user to choose from.
+If a banking app is not installed it will also display its AppStore link.
+The `BankSelectionBottomSheet` presentation requires a `PaymentComponentsController` instance from the `Integrate the Payment component` step 1.
+
+> **Note:** 
+> - The `BankSelectionBottomSheet` can be presented modally, used in a container view or pushed to a navigation view controller. Make sure to add your own navigational elements around the provided views.
```

Comment:
Clarification: The bottom sheet can be shown in a container view or pushed in a navigation view? Isn't a bottom sheet supposed only to be presented up from bottom?